tv_apps/smart-system/js/browser_context_menu.js

```diff
@@ -165,6 +165,26 @@
     this.scrollable.catchFocus();
   },
 
+  BrowserContextMenu.prototype._localizeElement = function(node, payload) {
```

Comment:
Since _localizeElement is the same as app_deck, how about move this function to `SharedUtils`.